src/common/crc32c_aarch64.c

```diff
@@ -96,6 +97,7 @@ uint32_t ceph_crc32c_aarch64(uint32_t crc, unsigned char const *buffer, unsigned
 
 	if (buffer) {
 #ifdef HAVE_ARMV8_CRYPTO
+	        if (ceph_arch_aarch64_pmull) {
```

Comment:
i think we should instead check this variable in `ceph_choose_crc32()`. i.e.:
```c
#elif defined(__arm__) || defined(__aarch64__)
  if (ceph_arch_aarch64_crc32 && ceph_arch_aarch64_pmull) {
    return ceph_crc32c_aarch64;
  }
```

Reply:
@tchaikov No. The check for crc32 is necessary to be able to use ceph_crc32c_aarch64. CRC opcode is a must.
PMULL allows further improvement, but it is optional.

Reply:
oic. thanks!